msp/compliance/orgs_and_cloud_accounts_report/orgs_and_cloud_accounts_report.pt

```diff
@@ -0,0 +1,278 @@
+      "query_params":{"view":"extended"}
+    }
+  EOF
+end
+
+script "js_filter_organizations", type: "javascript" do
+  parameters "ds_current_user_organizations", "param_exclude_organizations", "param_exclude_organizations_ids"
+  result "results"
+  code <<-EOF
+  if ( param_exclude_organizations.length > 0) {
+    var results = _.reject(ds_current_user_organizations, function(org){
+      return _.contains(param_exclude_organizations, org["name"])
+    })
+  } else if ( param_exclude_organizations_ids.length > 0) {
+    var results = _.reject(ds_current_user_organizations, function(org){
+      return _.contains(param_exclude_organizations_ids, org["org_id"])
+    })
+  }
+  EOF
+end
+
+script "js_generate_report", type: "javascript" do
+  parameters "ds_previous_six_month_costs","param_cost_metric","ds_report","param_graph_dimension","ds_currency_code","ds_currency_reference"
+  result "report"
+  code <<-EOS
+  EOS
+end
+
+# We exclude AWS account from the bill connection as it will later already be listed already in ds_aws_cloud_vendor_accounts
+script "js_bill_connects_without_aws_payer", type: "javascript" do
+  parameters "ds_bill_connects"
+  result "result"
+  code <<-EOF
+    var cloud_vendor_to_exclude = ["aws"]
+    var result = _.reject(ds_bill_connects, function(account){
+      return _.contains(cloud_vendor_to_exclude, account["cloud_vendor_id"])
+    })
+  EOF
+end
+
+script "js_combine_cloud_accounts", type: "javascript" do
+  parameters "ds_bill_connects_csp", "ds_bill_connects_without_aws_payer", "ds_aws_cloud_vendor_accounts"
+  result "result"
+  code <<-EOF
+    var result = ""
+    if ( ds_bill_connects_csp.length > 0 ) {
+      result = ds_bill_connects_csp.concat(ds_bill_connects_without_aws_payer)
+    }
+    else {
+      result = ds_bill_connects_without_aws_payer
+    }
+    result = result.concat(ds_aws_cloud_vendor_accounts)
+  EOF
+end
+
+script "js_normalize_cloud_vendor_ids", type: "javascript" do
+  parameters "ds_combined_cloud_accounts"
+  result "result"
+  code <<-EOF
+  function dedup(arr) {
+    var hashTable = {};
+    return arr.filter(function (el) {
+      var key = JSON.stringify(el);
+      var match = Boolean(hashTable[key]);
+      return (match ? false : hashTable[key] = true);
+    });
+  }
+  for (var i = 0; i < ds_combined_cloud_accounts.length; i++) {
+    ds_combined_cloud_accounts[i].id = ds_combined_cloud_accounts[i].id.replace(/^(aws|azure-ea|azure-mca-enterprise|google)-/g, "")
+  }
+  var result = dedup(ds_combined_cloud_accounts);
+  EOF
+end
+
+ policy "policy_scheduled_report" do
+  validate_each $ds_normalized_cloud_vendor_ids do
+  	summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Cloud Accounts Found"
+  	detail_template <<-EOS
+{{ len data }} Cloud Accounts have been generated from: {{ rs_project_name }} (ID: {{ rs_project_id }}) \n
+| Flexera Org ID | Cloud Vendor | Cloud Account ID | AWS Account Name |
```

Comment:
some sorting would be useful here.  I suggest by Org and cloud vendor.  you can use the underscore.js sortBy function.

Reply:
Done. Added sort